### PR TITLE
Chore: EE models to raw

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: apps/meteor/ee/server/models/raw/LivechatPriority.ts [hunk 1]
```diff
@@ -24,4 +23,41 @@ export class LivechatPriorityRaw extends BaseRaw<ILivechatPriority> implements I
 
 		return this.findOne(query, options);
 	}
+
```
Comment (murtaza98) [inline apps/meteor/ee/server/models/raw/LivechatPriority.ts:26]: Indexes are missing for this collection

File: apps/meteor/ee/server/models/raw/LivechatUnitMonitors.ts [hunk 2]
```diff
@@ -1,11 +1,45 @@
 import type { ILivechatMonitor } from '@rocket.chat/core-typings';
 import type { ILivechatUnitMonitorsModel } from '@rocket.chat/model-typings';
-import type { Db } from 'mongodb';
+import type { Db, DeleteResult, FindCursor, UpdateResult } from 'mongodb';
 
 import { BaseRaw } from '../../../../server/models/raw/BaseRaw';
 
+// Fix this to use the right typing
```
Comment (murtaza98) [inline apps/meteor/ee/server/models/raw/LivechatUnitMonitors.ts:7]: I believe we can remove this file now :) https://github.com/RocketChat/Rocket.Chat/blob/chore/ee-models-to-raw/apps/meteor/ee/app/models/server/models/LivechatUnitMonitors.js#L0-L1

File: apps/meteor/ee/app/models/server/index.js [hunk 3]
```diff
@@ -1,14 +1,8 @@
-import CannedResponse from './models/CannedResponse';
-import LivechatPriority from './models/LivechatPriority';
-import LivechatTag from './models/LivechatTag';
-import LivechatUnit from './models/LivechatUnit';
 import LivechatUnitMonitors from './models/LivechatUnitMonitors';
```
Comment (murtaza98) [inline apps/meteor/ee/app/models/server/index.js:1]: Do we still need this?

File: apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js [hunk 4]
```diff
@@ -236,7 +237,7 @@ export const updateInquiryQueuePriority = (roomId, priority) => {
 
 export const removePriorityFromRooms = (priorityId) => {
 	LivechatRooms.findOpenByPriorityId(priorityId).forEach((room) => {
-		updateInquiryQueuePriority(room._id);
+		Promise.await(updateInquiryQueuePriority(room._id));
```
Comment (murtaza98) [inline apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js:240]: If syncronization isn't required, then maybe we can benifit from Promise.all() here

File: apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js [hunk 5]
```diff
@@ -249,7 +250,7 @@ export const updatePriorityInquiries = (priority) => {
 
 	const { _id: priorityId } = priority;
 	LivechatRooms.findOpenByPriorityId(priorityId).forEach((room) => {
-		updateInquiryQueuePriority(room._id, priority);
+		Promise.await(updateInquiryQueuePriority(room._id, priority));
```
Comment (murtaza98) [inline apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js:253]: If syncronization isn't required, then maybe we can benifit from Promise.all() here

File: apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js [hunk 6]
```diff
@@ -236,7 +237,7 @@ export const updateInquiryQueuePriority = (roomId, priority) => {
 
 export const removePriorityFromRooms = (priorityId) => {
 	LivechatRooms.findOpenByPriorityId(priorityId).forEach((room) => {
```
Comment (murtaza98) [inline apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js:239]: There is not definition for `LivechatRooms.findOpenByPriorityId`

File: apps/meteor/ee/server/models/raw/LivechatUnit.ts [hunk 7]
```diff
@@ -1,5 +1,189 @@
...
+						$set: {
+							parentId: _id,
+							ancestors,
+						},
+					},
+				);
+			}),
+		);
+
+		return _.extend(record, { _id });
```
Comment (sampaiodiego) [inline apps/meteor/ee/server/models/raw/LivechatUnit.ts:187]: let's reduce `underscore` usage 😬 
```suggestion
		return { ...record, _id };
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Pull request closed without merging**